python/src/StationaryFunctionalCovarianceModel_doc.i.in

```diff
@@ -0,0 +1,49 @@
+%feature("docstring") OT::StationaryFunctionalCovarianceModel
+"Functional covariance function.
+
+Allows to define an 1-d stationary covariance model from a custom correlation function.
```

Comment:
```suggestion
Allows to define a stationary covariance model with one-dimensional output from a custom correlation function.
```
Also, is the "Allows to" prefix mandatory in docstrings? Otherwise I think it would be better to simply write "Define..."